src/coreComponents/mpiCommunications/MpiWrapper.cpp

```diff
@@ -182,10 +266,10 @@ int MpiWrapper::wait( MPI_Request * request, MPI_Status * status )
   return 0;
 }
 
-int MpiWrapper::waitany( int count, MPI_Request array_of_requests[], int * indx, MPI_Status * status )
+int MpiWrapper::waitany( int count, MPI_Request array_of_requests[], int * indx, MPI_Status array_of_statuses[] )
```

Comment:
of course...camelcase isn't really enforced?

Reply:
Think this is a holdover from when we were implementing the mpiwrapper in a way that mimics the MPI interface closely minus the MPI_ prefix. When we did the large camelCase restructuring across the codebase that changed in this file e.g. Comm_rank to commRank, so we're not mimicking the mpi interfaces as closely anymore (there is actually a comment in the wrapper.cpp referring to the old methodology which I'm removing in my followup PR).

Reply:
ok